lib/GetStream/StreamChat/Channel.php

```diff
@@ -1,34 +1,36 @@
 <?php
 
+declare(strict_types=0);
```

Comment:
Not sure if it's worth adding type hints when they are not strict

Reply:
@AnatolyRugalev I do not intend to introduce a breaking change now. I plan to enable it in the next major version.

Let's say that the type hints are just warnings for now.